### ISSUE TITLE
I can't see playlists or albums other than my own 

### ISSUE THREAD

**Issue by Ataquemos** [vfsfitvnm/ViMusic#1689]
### Steps to reproduce the bug

For example, if I go to an artist's profile, and try to enter an album, it will appear this

### Expected behavior

I guess I should be able to see the songs and singles on the albums

### Actual behavior

When entering any album or single, a non-loading screen appears with the legend "Unknown" at the top.

### Screenshots/Screen recordings

![Screenshot_20240630-055056](https://github.com/vfsfitvnm/ViMusic/assets/174202075/c52fb111-c5cd-4541-af64-6f6e4e0d8d06)

### ViMusic version

0.5.4

### Android version

Android 11

### Additional information

I really like this app, music should be free for everyone, I hope you can solve the problem.

**Comment by leggendd2** [comment]
Me too. I am just using ViMusic and have the same problem.
Latest release is released in 2022, I am supposing that the app is not anymore maintained.
If is not maintained please add a flag in readme...

**Comment by leggendd2** [comment]
Try out https://github.com/fast4x/RiMusic
Is a Fork of ViMusic with enhanced features and is MANTAINED

**Comment by chandreshx** [comment]
Yeah, same issue. Any playlist can not be opened it just crashes like this.

**Comment by Ataquemos** [comment]
Thank you all very much for taking the time to respond, GitHub is a
beautiful community
